tasks/system_probe.py

```diff
@@ -609,14 +609,21 @@ def generate_cgo_types(ctx, windows=is_windows):
 
     for f in def_files:
         fdir, file = os.path.split(f)
+        absolute_input_file = os.path.abspath(f)
         base, _ = os.path.splitext(file)
         with ctx.cd(fdir):
+            output_file = "{base}_{platform}.go".format(base=base, platform=platform)
             ctx.run(
-                "go tool cgo -godefs -- -fsigned-char {file} > {base}_{platform}.go".format(
-                    file=file, base=base, platform=platform
-                )
+                "go tool cgo -godefs -- -fsigned-char {file} > {output_file}".format(file=file, output_file=output_file)
```

Comment:
Instead of using `>` (not available on Windows), use `out_stream` parameter of `run` to pass the output file

Reply:
Fair point :) it was already there